pkg/slack/modals/launch/launch.go

```diff
@@ -8,15 +8,44 @@ import (
 	"github.com/openshift/ci-chat-bot/pkg/slack/modals"
 	"github.com/sirupsen/logrus"
 	"github.com/slack-go/slack"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
 	"net/http"
 	"strings"
 	"sync"
 )
 
-// Identifier is the view identifier for this modal
-const Identifier modals.Identifier = "launch"
-const Identifier2ndStep modals.Identifier = "launch2ndStep"
-const Identifier3rdStep modals.Identifier = "launch3ddStep"
+const (
+	IdentifierInitialView        modals.Identifier = "launch"
+	Identifier3rdStep            modals.Identifier = "launch3ddStep"
+	IdentifierPRInputView        modals.Identifier = "pr_input_view"
+	IdentifierFilterVersionView  modals.Identifier = "filter_version_view"
+	IdentifierRegisterLaunchMode modals.Identifier = "launch_mode_view"
+	IdentifierSelectVersion                        = "select_version"
+)
+
+const (
+	stableReleasesPrefix        = "4-stable"
+	launchFromPR                = "pr"
+	launchFromMajorMinor        = "major_minor"
+	launchFromStream            = "stream"
+	launchFromLatestBuild       = "latest_build"
+	launchFromReleaseController = "release_controller_version"
+	//launchFromCustom            = "custom" TODO - if necessary, add a custom build input field in version selection
+	launchPlatform       = "platform"
+	launchArchitecture   = "architecture"
+	launchParameters     = "parameters"
+	launchVersion        = "version"
+	launchStepContext    = "context"
+	defaultPlatform      = "aws"
+	defaultArchitecture  = "amd64"
+	launchMode           = "launch_mode"
+	launchModeVersion    = "version"
+	launchModePR         = "pr"
+	launchModePRKey      = "One or multiple PR-s"
```

Comment:
`PR-s` seems weird to me.  Can it not `PR's` or just `PR's`?

Reply:
Fixed